### ISSUE TITLE
Ariel does not allow out of source builds because of fesimple.cc

### ISSUE THREAD

**Issue by nmhamster** [sstsimulator/sst-elements#48]
```
g++ -O3 \
	-I../../.. -I/Users/sdhammo/Documents/Git/sst-github-repo  \
	-pthread -I/Users/sdhammo/Documents/Boost/1.54.0/openmpi/1.8.1/clang/5.1.0/include \
	-DBIGARRAY_MULTIPLIER=1 \
	-DTARGET_IA32E -DHOST_IA32E -fPIC -DTARGET_MAC \
	-I/Users/sdhammo/Documents/PIN/2.14/source/include/pin \
	-I/Users/sdhammo/Documents/PIN/2.14/ \
	-I/Users/sdhammo/Documents/PIN/2.14/extras/components/include \
	-I/Users/sdhammo/Documents/PIN/2.14/source/include/pin/gen/ \
	-I/Users/sdhammo/Documents/PIN/2.14/extras/xed-intel64/include \
	-fomit-frame-pointer -fno-stack-protector \
	-stdlib=libstdc++ \
	-o fesimple.o -c \
	frontend/simple/fesimple.cc
clang: error: no such file or directory: 'frontend/simple/fesimple.cc'
clang: error: no input files
```

**Comment by nmhamster** [comment]
Will need to be tested after #222 is merged into devel. This includes the use of Automake hooks to ensure tool builds happen correctly.

**Comment by nmhamster** [comment]
Fixed.
